[skip ci][DOCS] Update for percentage columns and totals row data vis options (#40504)
elastic/kibana: docs/visualize/datatable.asciidoc

*Per Page*:: This field controls the pagination of the table. The default value is ten rows per page.

> *Total function*:: This field controls the function used to calculate totals which can be toggled with the show total checkbox.

Reviewer: ```suggestion
*Total function*:: This field controls the function used to calculate totals that you can toggle with the **Show total** checkbox.
```

*Percentage column*:: Select a column to add a percentage based column on the same data.

Checkboxes are available to enable and disable the following behaviors:

*Show metrics for every bucket/level*:: Check this box to display the intermediate results for each bucket aggregation.
*Show partial rows*:: Check this box to display a row even when there is no result.

> *Show total*:: Check this box to display a row at the bottom of the table with each column's total value.

Reviewer: Change the formatting here so that the checkboxes are indented under the Percentage column.  It should look like this:

<img width="763" alt="Screen Shot 2019-07-08 at 11 22 35 AM" src="https://user-images.githubusercontent.com/33642766/60833276-c120b280-a172-11e9-8010-cbd259e57223.png">

Author: Discussed offline and came up with a different structure that better explains the options tab.
